### PR TITLE
Landing page feature

### PR DESCRIPTION
**Pull request by LuiWo**
Landing/SignIn page completed with responsiveness.

Extra pages ( for testing) to be deleted after review.

Proper routing to be set when needed pages and routing are done.

### CODE REVIEW

File: TG-Netflix/src/components/landing-page/SignIn.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    <>
+      <Container>
+        <LoginCard>
+          <h1>Welcome.</h1>
+          <h2>Log in and keep watching</h2>
+          <Form onSubmit={handleSubmit}>
+            <Input>
+              <label>
+                Email:
+                <input type="email" placeholder="test@hotmail.com"></input>
```
Comment (lextirajoh) [inline TG-Netflix/src/components/landing-page/SignIn.jsx:27]: Really minor, but you can make empty tags self closing.

**Comment by lextirajoh** [comment]
Hey Luisa,

For reference I'm using the Dutch version of the login form, as I cant seem to access an English version. I assume it's the same except for the language.

- I would say the main thing is to give the Login Card  some padding, particularly on the L/R sides. With the Netflix form, all the elements inside the card are aligned equally. You could add a padding property to the Login Card and remove any padding from the individual elements to have them aligned the same. Or add a container div withing the Login Card to control everything as a whole.

- Login button can then be 100% width.

- You can add the Netflix logo at the top left of the screen. There will be no navbar on this landing page.

There are differences with the font colors and the captions above the input fields, but I assume they are intentional to not make it look too identical. Maybe check with jens what is the best way to avoid issues? I have no idea about that.

Responsiveness is identical. nice job!

**Comment by barterkamp** [comment]
Hi Luisa,

I checked the css of the original site. In addition to the comment made by Lex i found that the black container has the following css styles:

background-color: #333 (with some opacity)
min-height: 660px
padding: 60px 60px 40px

The inputfield have the following styles:

border: 0;
border-radius: 4px;
height: 50px
width: 100%;
line-height: 50px;
padding: 16px 20px 0;

The background image can be downloaded by right clicking on the background and open in new window.

**Comment by lextirajoh** [comment]
Everything is functional but a few things:

- in the desktop version the footer gets rendered outside the viewport unnecessarily, because you're setting the Container to 100vh.
- you can add a wee more top margin to the netflix logo.

If you haven't done already, maybe good to check with Jens what you need to do to avoid issues with phishing and how closely it can mimic it.

File: TG-Netflix/src/components/landing-page/SignIn.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import {
+  Wrapper,
+  LoginBody,
```
Comment (lextirajoh) [inline TG-Netflix/src/components/landing-page/SignIn.jsx:5]: Remove loginbody

File: TG-Netflix/src/components/landing-page/SignIn.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      setTimeout(() => {
+        navigate("/discover");
+      }, 500),
+      console.log("I work");
+  };
+
+  return (
+    <>
+      <Wrapper>
+        <LoginBody>
```
Comment (lextirajoh) [inline TG-Netflix/src/components/landing-page/SignIn.jsx:28]: Remove loginbody

File: TG-Netflix/src/components/landing-page/SignIn.jsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+                New to CloneFlix? <a href="#">Sign up now</a>
+              </p>
+              <br />
+              <p>
+                This product is not the real thing. It is a project made by a
+                group of developers. <a href="#">Learn more</a>
+              </p>
+            </ExtraInfo>
+          </LoginCard>
+        </LoginBody>
```
Comment (lextirajoh) [inline TG-Netflix/src/components/landing-page/SignIn.jsx:63]: Remove loginbody

File: TG-Netflix/src/components/landing-page/style.jsx [hunk 5]
```diff
@@ -0,0 +1,225 @@
+import style from "styled-components";
+import bgImage from "./NetflixBg2.jpg";
+
+// Variables for Netflix' color pallet 
+// https://www.color-hex.com/color-palette/22942 
+
+export const Container = style.div`
+    background-color: rgb(0,0,0, 0.6);
+    height: 100%;
```
Comment (lextirajoh) [inline TG-Netflix/src/components/landing-page/style.jsx:9]: set height: 100vh

File: TG-Netflix/src/components/landing-page/style.jsx [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+
+svg {
+    width: 250px;
+    height: 50px;
+    fill:  rgba(219,0,0);
+}
+`;
+
+export const Wrapper = style.div`
+    min-height: 100vh;
```
Comment (lextirajoh) [inline TG-Netflix/src/components/landing-page/style.jsx:52]: remove min-height property

File: TG-Netflix/src/components/landing-page/style.jsx [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+    &:after{
+        content: "";
+        height: 200px;
+        margin: 0 auto 55px;
+        display: block;
+        @media only screen and (max-width: 700px){
+            height: 35px;
+        }
+    }
+`;
```
Comment (lextirajoh) [inline TG-Netflix/src/components/landing-page/style.jsx:78]: remove loginbody

File: TG-Netflix/src/components/landing-page/style.jsx [hunk 8]
```diff
@@ -0,0 +1,225 @@
...
+
+///////////Placeholder
+
+export const Footer = style.div`
+    background-color: pink;
+    display: block;
+    width: 100%;
+    height: 220px;
+    position: relative;
+    bottom: -10px;
```
Comment (lextirajoh) [inline TG-Netflix/src/components/landing-page/style.jsx:224]: replace lines 222-224 with:

position: absolute;
bottom: 0;
left: 0;

**Review by lextirajoh — COMMENTED** [review]
Push before making these changes, but for me it fixed it. Footer stays at the bottom and within the screen, both on mobile and desktop!

File: TG-Netflix/src/components/landing-page/style.jsx [hunk 9]
```diff
@@ -0,0 +1,225 @@
...
+    background-color: black;
+    }
+`;
+
+export const Background = style.div`
+background-image: url(${bgImage}); 
+background-repeat: no-repeat;
+background-position: center;
+background-size: cover;
+padding-bottom: 10px;
```
Comment (lextirajoh) [inline TG-Netflix/src/components/landing-page/style.jsx:20]: remove padding

**Review by lextirajoh — DISMISSED** [review]

**Review by lextirajoh — DISMISSED** [review]

**Review by Sander-G — DISMISSED** [review]
approve!

**Review by Sander-G — APPROVED** [review]
approve!!

**Pull request merged**